charts/merlin/values-global.yaml

```diff
@@ -1,6 +1,6 @@
 global:
   ingressIP: ""
-  oauthclient: "test-client-123"
+  oAuthClient: "test-client-123"
```

Comment:
Nit: if we are changing, why not change it to `oAuthClientId`

Reply:
`oAuthClientId` or `oAuthClientID` ?

Reply:
`oauthClientID` should be better.